src/card.css

```diff
@@ -0,0 +1,76 @@
+.card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+}
+  
+.card__picture {
+  width: 160px;
+  height: 134px;
+  display: block;
+  margin: 0 auto 40px;
+}
+  
+.card__discription {
+  margin-bottom: 4px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+}
+  
+.card__code {
+  margin-bottom: 16px;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
+  
+.card__cost {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin-bottom: 16px;
+}
+  
+.card__value {
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 18px;
+  color: #060b35;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+  
+.card__price {
+  font-size: 12px;
+  line-height: 18px;
+  color: #616070;
```

Comment:
Move repeatable colors to CSS variables.

[More info.](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)